integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/rest/test/header/ContentTypeTestCase.java

```diff
@@ -44,66 +47,72 @@
 import org.wso2.esb.integration.common.utils.ESBTestCaseUtils;
 
 public class ContentTypeTestCase extends ESBIntegrationTest {
+
     private Log log = LogFactory.getLog(ContentTypeTestCase.class);
-	private HttpServer server = null;
-	public static String contentType;
-
-	@BeforeClass(alwaysRun = true)
-	public void init() throws Exception {
-		super.init();
-		String relativePath = "/artifacts/ESB/synapseconfig/esbjava2283/api.xml";
-		ESBTestCaseUtils util = new ESBTestCaseUtils();
-		relativePath = relativePath.replaceAll("[\\\\/]", "/");
-		OMElement apiConfig = util.loadResource(relativePath);
-		addApi(apiConfig);
-	}
+    public static String contentType;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+
+        super.init();
+        String relativePath = "/artifacts/ESB/synapseconfig/esbjava2283/api.xml";
+        ESBTestCaseUtils util = new ESBTestCaseUtils();
+        relativePath = relativePath.replaceAll("[\\\\/]", "/");
```

Comment:
relativePath variable do not have any backslash. So, why we need to replace non existing characters in this line?